services/drupal/web/themes/epa_theme/templates/form/entity-moderation-form.html.twig

```diff
@@ -2,13 +2,57 @@
   {{ form.node_edit_link }}
 {% endif %}
 
-<ul class="entity-moderation-form">
-  <li class="entity-moderation-form__item">{{ form.current }}</li>
-  <li class="entity-moderation-form__item">{{ form.revision_author }}</li>
-  <li class="entity-moderation-form__item">{{ form.review_deadline }}</li>
-  <li class="entity-moderation-form__item">{{ form.new_state }}</li>
-  <li class="entity-moderation-form__item">{{ form.revision_log }}</li>
-  <li class="entity-moderation-form__item">{{ form.workflow_508_compliant }}</li>
-  <li class="entity-moderation-form__item">{{ form.submit }}</li>
-</ul>
-{{ form|without('current', 'revision_author', 'review_deadline', 'new_state', 'revision_log', 'submit', 'workflow_508_compliant', 'node_edit_link') }}
+<div class="box box--special">
+  <div class="box__content">
+    {% if form.nid %}
```

Comment:
I'm surprised we couldn't override the form field templates directly rather than doing this workaround.  Did you look into that @dcmouyard ?

Reply:
@azinck There aren’t any template suggestions for form items, so I went with this approach instead of creating new template suggestions.

Another approach would be to pipe these through the correct theme functions in `epa_workflow.module`, but I had already done the current approach before I learned where these items were set.

Reply:
I can live with this, thanks.